nix/modules/tezos-accuser.nix

```diff
@@ -0,0 +1,42 @@
+# SPDX-FileCopyrightText: 2021 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+{config, lib, pkgs, ...}:
+
+with lib;
+
+let
+  tezos-accuser-pkgs = {
+    "007-PsDELPH1" =
```

Comment:
Btw, we've dropped 007 support some time ago, let's drop it in these services as well :wink: